### DISCUSSION TITLE
query monitor

### DISCUSSION THREAD

**Discussion by icepicknz** [stuttter/ludicrousdb#202] [Help] [answered]
is there any way to run query monitor and ludicrusDB at the same time? i.e. including the other db.php drop in at the bottom of your db.php drop in?

**Comment by jonnynews** [comment]
@icepicknz This already works.

**Comment by icepicknz** [comment]
how?

root@shop1:/var/www/html/wp-content# ls -al db.php 
lrwxrwxrwx 1 root webdev 39 Nov 27 12:27 db.php -> plugins/query-monitor/wp-content/db.php

root@shop1:/var/www/html/wp-content# ls -al plugins/query-monitor/wp-content/db.php
-rw-rw-r-- 1 www-data www-data 2403 Nov 28 11:43 plugins/query-monitor/wp-content/db.php

root@shop1:/var/www/html/wp-content# ls -al ../db-config.php  
-rw-r--r-- 1 www-data www-data 4577 Dec 13 19:31 ../db-config.php

Since putting db.php there for query monitor, I can no longer alter additional read/write nodes in db-config.php to share traffic load.

do I need to include something in wp-config?

**Comment by jonnynews** [comment]
@icepicknz No you need the db.php from ludicrousdb in wp-content and it should work.

**Comment by icepicknz** [comment]
but then you loose all the debug part of query monitor

**Comment by jonnynews** [comment]
> but then you loose all the debug part of query monitor

I don't think you do. Can you give it a try and get back to me.

**Comment by JJJ** [comment, marked as answer]
In short, no.

The caveat with WordPress drop-in plugins (replacement classes & functions) is that there can only be 1 replacement at a time.

LudicrousDB could (and probably should) explicitly somehow have a dedicated integration for the awesome Query Monitor plugin.
